osprey.js

```diff
@@ -32,9 +32,12 @@ exports.addJsonSchema = function (schema, key) {
 exports.loadFile = function (path, opts) {
   var options = opts || {}
 
-  return require('raml-parser')
-    .loadFile(path)
-    .then(function (raml) {
+  return require('raml-1-parser')
+    .loadRAML(path)
+    .then(function (ramlApi) {
+      var raml = ramlApi.toJSON({
```

Comment:
Unfortunately, but the parser team is working on that - see https://github.com/raml-org/raml-js-parser-2/issues/371.